libs/sdk-ui-dashboard/src/model/commandHandlers/ui/changeRenderModeHandler.ts

```diff
@@ -15,7 +16,12 @@ export function* changeRenderModeHandler(
         correlationId,
     } = cmd;
 
-    yield put(uiActions.setRenderMode(renderMode));
+    const editModeEnabled = yield select(selectDashboardEditModeDevRollout);
 
-    return renderModeChanged(ctx, renderMode, correlationId);
+    if (renderMode == "view" || editModeEnabled) {
```

Comment:
Nitpick: please use triple equals if possible, ESLint should be complaining about this 🤔

Reply:
Also it should maybe be !== ?

Reply:
I didn't have eslint setup so it didn't warn me. It checks if target render mode is view, so it should be ===.